mu4/notation/inotationselection.h

```diff
@@ -36,6 +36,8 @@ class INotationSelection
     virtual Element* element() const = 0;
     virtual std::vector<Element*> elements() const = 0;
 
+    virtual int firstStaffIndex() const = 0;
```

Comment:
Better not add that method to this interface. This is an abstract class for a selection. it shouldn't know about staves

Reply:
But ... it's not that simple. In this case, staffIndex is the index of staff inside System. And libmscore expects this index to work correctly. The selection can be started from the first staff or from any other.

Reply:
please rename this function to something like "startStaffIndex"